File: CSharp/Math/IntOverflow.cs
using static System.Console;

public class Program {
	public static void Main() {
		for (int i = 1; i >= 0; i *= 2) { WriteLine($"i: {i}"); }
		checked {
			for (int j = 1; j >= 0; j *= 2) { WriteLine($"j: {j}"); }
		}
	}
}

//https://pt.stackoverflow.com/q/326539/101

File: Java/Math/IntOverflow.java
class Loop {
	public static void main(String[] a) {
		int cont = 0;
		for (int i = 1; i < 10000000000000000L; i *= 2, cont++) { System.out.println("i:" + i); }
		System.out.println("cont:"+cont);
	}
}

//https://pt.stackoverflow.com/q/326539/101
